enclave/core/sgx/asmcommon.inc

```diff
@@ -28,13 +28,22 @@
     push %rax
     push %rdx
 
+    // Check if XSAVE/XRSTOR is supported
+    lea     oe_is_xsave_supported(%rip), %rax
```

Comment:
Perhaps this can be `mov oe_is_xsave_supported(%rip),  %eax`

Reply:
Thanks, removed the extra `lea`.